src/content/configuration/cache.mdx

```diff
@@ -109,6 +110,25 @@ module.exports = {
 };
 ```
 
+### cache.cacheUnaffected
+
+Cache computation of modules which are unchanged and reference only unchanged modules. It can only be used along with [`cache.type`](#cachetype) of `'memory'`, besides, [`experiments.cacheUnaffected`](/configuration/experiments/#experimentscacheunaffected) must be enabled to use it.
+
+- Type: `boolean`
+- v5.54.0+
```

Comment:
Not using `<Badge text="v5.34.0+" />` ?

Reply:
Too distracting when they are many badges on one page. Body text serves the purpose well.